### ISSUE TITLE
Add lua method to release held key

### ISSUE THREAD

**Issue by MaslovKK** [Facepunch/garrysmod-requests#2226]
For example: Player helds key, then clicks on button that releases key, and this key now has no effect until this key will be physically released and will be down again

**Comment by MaslovKK** [comment]
> This is already possible with https://wiki.facepunch.com/gmod/CUserCmd:RemoveKey

_key now has no effect until this key will be physically released and will be down again_

**Comment by robotboy655** [comment]
Yeah, you can code a simple set-reset latch. You would simply keep removing the key until the key is depressed.
